### PR TITLE
Docs: Update IconButton styling and size across docs

### PR DESCRIPTION
**Pull request by ayeshakmaz**
JIRA[ PDS-2499](https://jira.pinadmin.com/browse/PDS-2499)

- Swap expand/collapse icons (both the chevron next to props and the eye next to code blocks) with maximize/minimize icons.
- Swap the copy code icon (currently using the Story Pin icon) with the drag-drop icon.
- Update Icon Button size next to section titles from "s" to "xs" (so that all icons are consistently sized at 24x24)
- Update all Icon Button colors to "darkGray" instead of "gray"
- Ensure that every icon provides a Tooltip on hover. Make sure the Tooltip text is in sentence case.
No tooltip → "Copy link"
"Expand/Collapse Props" → "Expand/Collapse props"

## Test Plan

current docs tests

### CODE REVIEW

File: docs/src/components/Card.js [hunk 1]
```diff
@@ -58,20 +58,22 @@ export default function Card({
...
+                  dangerouslySetSvgPath={{
+                    __path:
+                      'M21.001 7.241l-4.053 4.052-1.06-1.06.672-.672a1.5 1.5 0 10-2.121-2.121l-.671.672-1.061-1.06L16.759 3l4.242 4.241zm-9.708 9.708l-4.052 4.052-4.242-4.241 4.053-4.053 1.059 1.06-.671.672a1.5 1.5 0 002.121 2.121l.671-.672 1.061 1.061zM14.639.879l-4.053 4.052a3 3 0 000 4.242l1.061 1.06-1.415 1.414-1.06-1.061a3 3 0 00-4.241 0L.879 14.638a2.998 2.998 0 000 4.242l4.241 4.242a3 3 0 004.241 0l4.053-4.052a3 3 0 000-4.242l-1.06-1.061 1.414-1.413 1.06 1.06a3 3 0 004.241 0l4.052-4.052a2.998 2.998 0 000-4.242L18.88.879a2.997 2.997 0 00-4.241 0z',
+                  }}
+                  accessibilityLabel={`${name} - Anchor tag`}
+                  onClick={() => {
+                    copyToClipboard(slugifiedId);
+                  }}
+                  size="xs"
+                  href={`#${slugifiedId}`}
```
Comment (dangerismycat) [inline docs/src/components/Card.js:72]: While you're in here, do you mind removing the `href` and adjusting the `role`? We probably want a better a11y label as well

File: docs/src/components/MainSectionSubsection.js [hunk 2]
```diff
@@ -29,19 +29,22 @@ const MainSectionSubsection = ({ children, description, title }: Props): Node =>
...
-            />
+            <Tooltip inline text="Copy link">
+              <IconButton
+                dangerouslySetSvgPath={{
+                  __path:
+                    'M21.001 7.241l-4.053 4.052-1.06-1.06.672-.672a1.5 1.5 0 10-2.121-2.121l-.671.672-1.061-1.06L16.759 3l4.242 4.241zm-9.708 9.708l-4.052 4.052-4.242-4.241 4.053-4.053 1.059 1.06-.671.672a1.5 1.5 0 002.121 2.121l.671-.672 1.061 1.061zM14.639.879l-4.053 4.052a3 3 0 000 4.242l1.061 1.06-1.415 1.414-1.06-1.061a3 3 0 00-4.241 0L.879 14.638a2.998 2.998 0 000 4.242l4.241 4.242a3 3 0 004.241 0l4.053-4.052a3 3 0 000-4.242l-1.06-1.061 1.414-1.413 1.06 1.06a3 3 0 004.241 0l4.052-4.052a2.998 2.998 0 000-4.242L18.88.879a2.997 2.997 0 00-4.241 0z',
+                }}
+                accessibilityLabel={`${title} - Anchor tag`}
+                size="xs"
+                href={`#${slugifiedId}`}
```
Comment (dangerismycat) [inline docs/src/components/MainSectionSubsection.js:40]: ditto here

**Review by dangerismycat — APPROVED** [review]
Looks great! Feel free to ignore my request if you just want to merge this, it would just be saving me a tiny bit of work 😅

**Comment by AlbertCarreras** [comment]
the icons are in the baseline, should we center them?

image

File: docs/src/components/Card.js (hunk 1 above)
Comment (dangerismycat) [reply docs/src/components/Card.js]: thanks! 🙂

**Review by dangerismycat — APPROVED** [review]

**Review by AlbertCarreras — APPROVED** [review]

**Pull request merged**